server/models/schema.sql

```diff
@@ -764,6 +764,8 @@ CREATE TABLE `inventory` (
   `stock_min` INT(10) UNSIGNED NOT NULL DEFAULT 0,
   `type_id` TINYINT(3) UNSIGNED NOT NULL DEFAULT 0,
   `consumable` TINYINT(1) NOT NULL DEFAULT 0,
+  `is_brocken` TINYINT(1)  NULL DEFAULT 0,
```

Comment:
This should be NOT NULL with a DEFAULT, to make sure that the default is always used.